### ISSUE TITLE
feature: Telegram Webhook

### ISSUE THREAD

**Issue by NovaLuvsPython** [Smug246/Luna-Grabber#766]
### What Feature?

**Telegram Option**
Instead of sending the info to discord make a second option (telegram) people can choose.
Ive been banned on 4 accs on discord within 1 week because of malware, but when i tried using Blanks grabber with tg its still lasting with 1 month without term.

so please add it many people would like that. 😎

### Check the following before submitting (If Applicable)

- [X] I have checked the existing [issues](https://github.com/Smug246/Luna-Grabber/issues)
- [ ] I want to work on this feature

**Comment by Smug246** [comment]
don't talk about it on discord then lol. discord won't ban you because of webhooks. telegram is on the 'to do list'

**Comment by Jokke999** [comment]
> don't talk about it on discord then lol. discord won't ban you because of webhooks. telegram is on the 'to do list'

Yes they do. Maybe they didnt before, but recently i got banned on multiple accounts. Accounts i never used for anything but grabbers sending to webhook. I suspect this is a new thing discord are doing, havent had the problem before.

**Comment by Smug246** [comment]
thats never happened to me and i test loads of shi on my account. maybe its because u dont have 2fa or an email or phone number they flag your account.
